solutions/swb-app/src/accountRoutes.ts

```diff
@@ -3,12 +3,15 @@
  *  SPDX-License-Identifier: Apache-2.0
  */
 
-// AWS Account management
 import {
   CreateAccountSchema,
   UpdateAccountSchema,
   HostingAccountService
 } from '@aws/workbench-core-accounts';
+import {
+  ListAccountRequest,
+  ListAccountsRequestParser
+} from '@aws/workbench-core-accounts/lib/models/accounts/listAccountsRequest';
```

Comment:
should these be added to the index file of `workbench-core/accounts` so the import isnt as clunky?